### PR TITLE
perf(wfs): Avoid creating needless objects

### PR DESCRIPTION
**Pull request by gliegard**
## Description
Fix performances issues introduced in #519

## Motivation and Context
In the example globe_wfs_extruded, we can see that it's slow.

It's because, #519 introduced a lot of objects creation needlessly.

@autra explained that here https://github.com/iTowns/itowns/pull/519#pullrequestreview-72819496

### CODE REVIEW

File: src/Renderer/ThreeExtended/Feature2Mesh.js [hunk 1]
```diff
@@ -121,41 +120,49 @@ function addFaces(indices, length, offset) {
...
+    let offset = 0;
 
     /* eslint-disable guard-for-in */
     for (const id in coordinates.featureVertices) {
         const { contour, property } = extractFeature(coordinates, properties, id);
         // get altitude from properties
         const altitude = getAltitude(options, property);
         const newVertices = coordinatesToVertices(contour, altitude);
-        vertices = vertices.concat(newVertices);
+        for (let i = 0; i < newVertices.length; i++) {
```
Comment (peppsac) [inline src/Renderer/ThreeExtended/Feature2Mesh.js]: You should instead change `coordinatesToVertices` signature and add an `target` optionnal parameter.

Then use it like this: 
```js
coordinatesToVertices(contour, altitude, new Float32Array(vertices.buffer, offset * 4));
```

Comment (peppsac) [reply src/Renderer/ThreeExtended/Feature2Mesh.js]: (Or pass the target and offset as arguments if you prefer)

**Comment by gliegard** [comment]
updated

File: src/Renderer/ThreeExtended/Feature2Mesh.js (hunk 1 above)
Comment (gliegard) [reply src/Renderer/ThreeExtended/Feature2Mesh.js]: thanks for your suggestion, I've updated the code !

**Comment by gchoqueux** [comment]
@gliegard I rebased the branch

**Comment by gliegard** [comment]
@itowns-reviewers

**Review by peppsac — APPROVED** [review]
Looks good, thanks.

**Pull request merged**